web/src/components/WikipediaBrowser/WikipediaBrowser.tsx

```diff
@@ -60,7 +60,11 @@ const WikipediaBrowser: React.FC = () => {
       e.stopPropagation();
       window.open(image.src, "_blank");
     } else {
-      if (link instanceof HTMLAnchorElement && link.href && link.title) {
+      if (link?.classList[0] === 'extiw') { 
```

Comment:
This would be a good place for a comment to explain why we care about `extiw` specifically.

I'm hesitant to use class names, as they can change, but we may not have a better option for this.